Makefile

```diff
@@ -126,10 +126,6 @@ CORE_COPYRIGHT_WWW?=	https://www.deciso.com/
 CORE_COPYRIGHT_YEARS?=	2014-2023
 
 CORE_DEPENDS_amd64?=	beep \
-			py${CORE_PYTHON}-duckdb \
```

Comment:
not sure which CORE_ARCH is exposed on your end, but we could move them to this CORE_DEPENDS_xxx variant and then inline this variant into CORE_DEPENDS_amd64